client/src/pages/Configurations/ConfigurationTitleBar.tsx

```diff
@@ -84,8 +95,25 @@ export function ConfigurationTitleBar({ step }: ConfigurationTitleBarProps) {
             </div>
           </div>
         </div>
-        <p> {CONFIGURATION_TITLE_CONTENTS[step].subtitle}</p>
+        <p>
+          {boldWord(
+            CONFIGURATION_TITLE_CONTENTS[step].subtitle,
+            CONFIGURATION_TITLE_CONTENTS[step]?.boldedWords ?? undefined
+          )}
```

Comment:
I think we can make this a bit simpler if we change `subtitle` to type `React.ReactNode` because then we could do something like this:
```ts
    activate: {
      title: 'Turn on configuration',
      subtitle: (
        <span>
          Refiner will <span className="font-bold">immediately</span> start to
          refine eCRs with {condition} as a reportable condition, in accordance
          with this configuration.
        </span>
      ),
```
And this line would become:
```suggestion
{CONFIGURATION_TITLE_CONTENTS[step].subtitle}
```

No logic needed 👍